builder/builder-next/executor_unix.go

```diff
@@ -28,7 +28,7 @@ func newExecutor(root, cgroupParent string, net libnetwork.NetworkController, ro
 	}
 	return runcexecutor.New(runcexecutor.Opt{
 		Root:                filepath.Join(root, "executor"),
-		CommandCandidates:   []string{"runc"},
+		CommandCandidates:   []string{"runu"},
```

Comment:
please include the errors (messages) which we need to modify this line. does `docker run --runtime` doesn't help this ?

Reply:
The error was as the commit message suggest: failed to find runc binary

Reply:
But since its a slice of strings, I have done something like this for the time being, `[]string{"runu", "runc"}`

Reply:
Did u add daemon.json when use --runtime option ?

Reply:
What I was trying to say is that, the dockerd terminates with an error message that it isn't able to find the runu binary. So we aren't able to move forward enough to run the docker client and pass a argument like --runtime in to it.

Reply:
so did you use daemon.json having run definition but still have this issue ?
it would be nice to avoid hard-coded value (e.g., defined from config file/daemon.json).

Reply:
> daemon.json having run definition but still have this issue ?

Yes, I am using this, but still the error pops up. I think the reason would be that it has hardcoded value to find runc so it is looking for runc executable?